File: 1281. Subtract the Product and Sum of Digits of an Integer/Solution.js
/**
 * @param {number} n
 * @return {number}
 */
var subtractProductAndSum = function(n) {
    let sum = 0;
    let product = 1;
    while(n>0){
        let num = n%10;
        sum = sum + num;
        product = product * num;
        n = Math.floor(n / 10)
    }
    return product - sum;
};
console.log(subtractProductAndSum(234));

File: 1281. Subtract the Product and Sum of Digits of an Integer/Solution.java
public class Solution {

    public int subtractProductAndSum(int n) {
        int answer;
        int product = 1;
        int sum = 0;

        while (n>0) {
            int num = n%10;
            product=num*product;
            sum = sum + num;
            n = n/10;  
        }
        answer = product - sum;

        
        return answer;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int answer = solution.subtractProductAndSum(234);
        System.out.println(answer);
    }
    
}